### PR TITLE
add distro specific branches to s2i scripts

### PR DESCRIPTION
**Pull request by pkubatrh**
also move rhel8 varnish-6 image to ubi8/s2i-core

### CODE REVIEW

**Comment by pkubatrh** [comment]
[test-openshift]

File: src/s2i/bin/run [hunk 1]
```diff
@@ -26,6 +26,12 @@ else
   exit 3
 fi
 
+if [ ${PLATFORM} == "el7" ]; then
+    VARNISH_VCL=/etc/opt/rh/rh-varnish{{ spec.version }}/varnish/default.vcl
```
Comment (hhorak) [inline src/s2i/bin/run]: Wouldn't it be better to put this path into `multispec.yml` and then define `VARNISH_VCL` in the Dockerfile? I'm not that experienced with dist-gen, so not sure entirely about this.

Comment (pkubatrh) [reply src/s2i/bin/run]: Makes sense. I moved the definition of `VARNISH_VCL` to Dockerfiles.

**Comment by pkubatrh** [comment]
Tests are green and diff looks sane, merging.

**Pull request merged**